man/passwd.1.xml

```diff
@@ -94,23 +94,16 @@
       </para>
 
       <para>
-	Then, the password is tested for complexity. As a general guideline,
-	passwords should consist of 6 to 8 characters including one or more
-	characters from each of the following sets:
+	Then, the password is tested for length.  As a general guideline,
+	passwords should be long.  It's fine to use simple character sets,
+	such as passwords consisting only of lowercase letters.  For a
```

Comment:
this statement is much much too weak.  yes, it follows a "general guideline that passwords should be long", but this sentence feels disconnected from that and reads as independent guidance.

Reply:
True.  Maybe rephrased as: `It's fine to use simple character sets,
	such as passwords consisting only of lowercase letters` + ` if it helps memorize longer passwords`.